geomesa-spark/geomesa-spark-jts/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@metasim Oops, this one too. Sorry I missed it on the first pass

Reply:
I suggest keeping this one in there, as it reduces the amount of noise you get when running the tests. Even though the tests may not explicitly log, Spark sure does....